core/src/main/resources/reference.conf

```diff
@@ -249,6 +249,9 @@ filesystems {
   s3 {
     class = "cromwell.filesystems.s3.S3PathBuilderFactory"
   }
+  http {
+    class = "cromwell.filesystems.http.HttpPathBuilderFactory"
```

Comment:
Given that this is in `reference.conf` do we want a way to turn this off that doesn't equal "sabotage with an invalid class path"?

Reply:
Yeah that's a good point.

Reply:
Also discussed in person, this is defining a filesystem named `http` but isn't configuring the default `Local` backend to use it. So using the HTTP filesystem would be opt-in for the default Local backend which is what we want.

Reply:
Adding a FS here makes it "available for referencing" but they still need to be explicitly referenced it in the engine or backend `filesystems` stanza to really be usable

Reply:
Per discussion yesterday we decided the http filesystem should be on by default in the default `Local` backend but that it should also be possible to turn it off. I've added code that allows setting `-Dbackend.providers.Local.config.filesystems.http.enabled=false` to turn the http filesystem off in `Local`.